### ISSUE TITLE
gzip when making requests?

### ISSUE THREAD

**Issue by dimitry12** [Unicity/Gauntlet#36]
Hello, @powerc9000 ! It's a nice tool you are building here!

Does it use gzip when making requests? If yes, how can I confirm? In either case, it would be cool to control that for each test: so that we can compare performance with and without compression on the application-level protocol.

**Comment by dimitry12** [comment]
Hello? @powerc9000

**Comment by powerc9000** [comment]
I don't think it gzips. I think it _can_ gzip. I don't have anything built in to toggle that functionality currently.
